piff/stats.py

```diff
@@ -580,7 +580,10 @@ def compute(self, psf, stars, logger=None):
         self.u = positions[mask, 0]
         self.v = positions[mask, 1]
         self.flux = shapes_truth[mask, 0]
-        self.reserve = np.zeros_like(self.u, dtype=bool)  # For now, all reserve = False
+        self.reserve = np.zeros_like(self.u, dtype=bool)
+        for i,s in enumerate(stars):
+            if s.is_reserve:
```

Comment:
I like `self.reserve[i] = s.is_reserve` here, or maybe even `self.reserve = np.array([s.is_reserve for s in stars], dtype=bool)`